### PR TITLE
Vagrant setup

### PR DESCRIPTION
**Pull request by larroy**

### CODE REVIEW

**Comment by kyrias** [comment]
Hm, wouldn't it make more sense to have this use the git repo by default, and then optionally supporting the latest release?

File: scripts/vagrant/provision.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+set -x
+sudo apt-get -y install build-essential python2.7-dev libffi-dev \
+  python-pip python-setuptools sqlite3 \
+  libssl-dev python-virtualenv libjpeg-dev libxslt1-dev apache2
+
+cd /var/www/html/
+wget -O - "https://vector.im/packages/vector-v0.8.2.tar.gz" | sudo tar xvvzf -
```
Comment (erikjohnston) [inline scripts/vagrant/provision.sh]: We probably want to get the latest version rather than a fixed one?

**Comment by erikjohnston** [comment]
Thanks for this, sorry for dropping it on the floor.

**Comment by PMaynard** [comment]
I think it would be better to use ansible instead of a bash script as it is more efficient and reusable and is supported by vagrant (ansible_local is the best option).

There is already [this](https://github.com/EMnify/matrix-synapse-auto-deploy), which is mentioned in the matrix docs/faq (I think). 

I have also made a similar one at <https://github.com/matrix-ac/matrix-conf/blob/master/roles/synapse/tasks/main.yml>.

**Comment by erikjohnston** [comment]
I'm not really familiar with any of this tech, so I'm going to close this for now. People should feel free to pick this up finish it off and I'll be happy to look at it again :)

**Pull request closed without merging**